csvToJson/csvToJson.js

```diff
@@ -0,0 +1,29 @@
+// here I will bring in all the classes and create one common method to call all the classes
+const csvHandler = require('./Parser/FileParser.js');
+const jsonHandler = require('./JsonHandler/jsonHandler.js');
+
+
+class csvToJson{
+    constructor(csvFileLocation, jsonFileLocation){
+        this.csvFileLocation = csvFileLocation;
+        this.jsonFileLocation = jsonFileLocation;
+    }
+
+    // function to convert the csv file to a json object
+    convertCsvToJson(){
+        // csvHandler
+        const csvhandler = new csvHandler(this.csvFileLocation);
+        console.log("success");
```

Comment:
I can remove these when time comes for final production. I think we should keep them for error checking in the future.

Reply:
I think that is a good idea.  It would definitely help.

Reply:
good to know! I thought so too